examples/qwen2audio/run.py

```diff
@@ -510,27 +508,16 @@ def qwen_infer(self,
             text_to_overwrite = new_token_positions[batch_indices,
```

Comment:
text_to_overwrite can be removed then?

Reply:
Good catch, I was able to remove much more dead code 😁